dags/atd_executive_dashboard_row_weekly_summary.py

```diff
@@ -179,4 +181,17 @@
         retry_delay=duration(seconds=60),
     )
 
-    t1 >> t2 >> t3 >> t4 >> t5 >> t6
+    t7 = DockerOperator(
+        task_id="ex_permits_issued",
+        image=docker_image,
+        auto_remove=True,
+        command=f"python AMANDA/amanda_to_s3.py --query ex_permits_issued",
+        environment=env_vars,
+        tty=True,
+        force_pull=False,
+        mount_tmp_dir=False,
+        retries=3,
+        retry_delay=duration(seconds=60),
+    )
+
+    t1 >> t2 >> t3 >> t4 >> t5 >> t6 >> t7
```

Comment:
@Charlie-Henry i can't remember if we've talked about this, or if it is even desired in this case, but it is possible to use [trigger rules](https://airflow.apache.org/docs/apache-airflow/1.10.10/concepts.html#trigger-rules) to enable downstream tasks to run regardless of if an earlier task fails.

Reply:
ah yes you did comment this before, good idea. I'll see if I can get something to work